modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -100,26 +230,34 @@ class ImagingUploader extends React.Component {
           </div>
           <div id="mri_upload_table">
             <StaticDataTable
-              Data={this.state.Data.Data}
-              Headers={this.state.Data.Headers}
-              getFormattedCell={formatColumn}
+              Data={this.state.data.Data}
+              Headers={this.state.data.Headers}
+              {
+                /* FIXME: Remove comments arround following line of code as soon
+                  as hiddenHeaders is accepted as a prop by the STaticDataTable
+                  Component.
+                */
+              }
+              {/*hiddenHeader={this.state.data.hiddenHeaders*/}
```

Comment:
hiddenHeaders are being set but they're not being used? I can't see any reference to hiddenHeaders in the old code on the left. am i missing it? if not, is there a reason why we are adding them in the refactoring?